Packages that will be used in the OHW19

ocefpaf opened the pull request:
Follow up from #372. This PR adds the extra packages that we will be using in the OHW19. Please let me know if this is the right file we should be modifying.

ocefpaf committed:
packages that will be used in the OHW19

ocefpaf commented:
Ping @rabernat and @tjcrone.

ocefpaf committed:
add podaacpy

ocefpaf commented on deployments/ocean/image/binder/environment.yml:
```diff
@@ -18,6 +18,7 @@ dependencies:
   - owslib
   - palettable
   - pocean-core
+  - podaacpy
```
Ping @emiliom.

tjcrone commented:
@ocefpaf, this looks great. I talked to Joe and he said we are fine to merge to staging during the Pangeo conference. One question, did you confirm that all of these packages are not already installed?

tjcrone commented:
A quick check seemed to indicate that owslib might already be there? Might be best not to install again?

ocefpaf committed:
rebase

ocefpaf committed:
remove owslib

ocefpaf commented:
> One question, did you confirm that all of these packages are not already installed?

I did but as you noticed I overlooked `owslib`. Just rebased against the latest commit on `staging` and removed `owslib`.
